templates/partials/content-post.php

```diff
@@ -4,6 +4,7 @@
     <header class="page-header">
       <h1><?php the_title() ?></h1>
       <div class="entry-meta date">
+        <?php the_author() ?>,
```

Comment:
Yeah @demotive and I started doing that and got into more CSS hell.

This will break if we do multiple authors anyway so my vote would be merge
this without.
On 16 Oct 2015 12:35 pm, "Dominic Baggott" <notifications@github.com> wrote:

> In templates/partials/content-post.php
> <https://github.com/nhsalpha/nhsblog/pull/1#discussion_r42233116>:
>
> > @@ -4,6 +4,7 @@
> >      <header class="page-header">
> >        <h1><?php the_title() ?></h1>
> >        <div class="entry-meta date">
> > +        <?php the_author() ?>,
>
> I think we should make it a link to the author page, eg
> http://digital.nhs.uk/author/joe-mcgrath
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nhsalpha/nhsblog/pull/1/files#r42233116>.
>